tests/Unit/TemplateRendererTest.php

```diff
@@ -27,17 +27,43 @@ public function testTemplateRendererInstance($renderer, $data, $supportedFeedTyp
     }
 
     /**
-     * Test that the template renderer renders a string.
+     * Test that the template renderer renders a string for the single-file template.
      *
      * @dataProvider templateRendererProvider
      * @return void
      */
     public function testRenderString($renderer, $data, $supportedFeedTypes)
     {
-        $this->assertInternalType(
-            "string",
-            $renderer->renderSimpleDatasetSite($data, $supportedFeedTypes)
-        );
+        // assertInternalType has been deprecated by the newer PHPUnit, however it is still required here for when the tests run in PHP 5.6 / PHPUnit 5.7
+        $outputHtml = $renderer->renderSimpleDatasetSite($data, $supportedFeedTypes);
+        if (method_exists($this,'assertIsString')) {
```

Comment:
You shouldn't put logic in tests, maybe split up the data provider into the `assertIsString` and `assertInternalType` tests instead